ESP8266wifi.h

```diff
@@ -57,6 +57,20 @@ struct Flags   // 1 byte value (on a system where 8 bits is a byte
          connectToServerUsingTCP:1;
 };
 
+enum listApTypes {WIFI_OPEN = 0,
```

Comment:
We might think about putting all datastructures and enums into the classes scope. However, this would require further adaptations to the examples etc. - and would be a little harder to use for beginners.